### DISCUSSION TITLE
Media specification mismatch

### DISCUSSION THREAD

**Discussion by sudhagar-mr-17051** [mastodon/mastodon#30481] [Q&A]
Hi Team,
While making calls to instance API, we could able to obtains server media configurations such as image size, image_matrix_limit, etc., but those configurations are not followed by the servers, They are allowing some size in API and while uploading we are getting an error. Incase of matrix limit also, we are facing same.

Kindly check this and provide us some solution.
Those who are using API frameworks are worried that the limitations are incorrect and couldn't met them with prevaldiations in our layer itself.

Thanks in advance

Regards,
Sudhagar M

@renchap @Gargron @andypiper

**Comment by andypiper** [comment]
Do not tag individual team members please.

“those configurations are not followed by the servers“ please provide specific examples. Which servers, what are the API limits reported, what is your reproducible test case? Have you contacted the server owners?
